packages/frontend-v2/utils/component_styles/buttons.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentStyleConfig } from "@chakra-ui/react";
+
```

Comment:
i wasn't sure if i should've created custom button components, but it seems like the best practice for chakra component styling looks something like this according to the docs:

https://chakra-ui.com/docs/styled-system/theming/component-style
https://chakra-ui.com/docs/styled-system/theming/customize-theme

i decided to go with this design since i'm not too sure what we want to do regarding component styling reuse. i think this is subject to change depending on what Aryan decides in the chakra design docs, but if anyone else has any thoughts pls lmk : D

Reply:
This is awesome! For purely stylistic components like Buttons we def can use this configuration like customization that allows us to create our own UI library/design system.